src/rpc/format/cbor/convert.rs

```diff
@@ -12,13 +12,15 @@ use surrealdb::sql::Thing;
 use surrealdb::sql::Uuid;
 use surrealdb::sql::Value;
 
-const TAG_DATETIME: u64 = 0;
+const TAG_STRING_DATETIME: u64 = 0;
+const TAG_DATETIME: u64 = 1;
 const TAG_NONE: u64 = 6;
-const TAG_UUID: u64 = 7;
+const TAG_STRING_UUID: u64 = 7;
```

Comment:
```suggestion
```

Reply:
In fact, let's support deserialising string based UUIDs, just in case. But we use binary UUIDs by default.